### ISSUE TITLE
Invalid API key

### ISSUE THREAD

**Issue by gary-reyes** [snjoetw/py-august#6]
As indicated in https://github.com/home-assistant/home-assistant/issues/15880 the API seems to be expired. A key that appears to work is listed below. I found it at https://github.com/jmaxxz/keymaker/issues/5

```
79fd0eb6-381d-4adf-95a0-47721289d1d9
```

**Comment by bradleyscott** [comment]
Thanks for the quick response @snjoetw. Has this released been pushed to pypi? Or will it just take them some time to report the new version?

**Comment by snjoetw** [comment]
Not released yet as I ran into issues yesterday when publishing. I’ll try again tonight.

**Comment by ramginuga** [comment]
great library. is the fix released?

**Comment by snjoetw** [comment]
yes i've released the lib and integrated with HASS: https://github.com/home-assistant/home-assistant/pull/15916
